app/reducers/tabs.js

```diff
@@ -443,13 +443,13 @@ export default function tabs( state: array = initialState, action )
         }
         case UI_TYPES.RESET_STORE :
```

Comment:
@hunterlester let's not change this behaviour by default. Otherwise `reset store` doesn't do what you expect.